general-development/d-solid-en.json5

```diff
@@ -0,0 +1,23 @@
+{
+  type: 'choose_options',
+  lang: 'en',
+  tags: ['development', 'patterns'],
+  card: {
+    question: 'What is `D` in SOLID?',
+    answers: [
+      { text: 'Development' },
+      { text: 'Domain' },
+      { text: 'Data driven' },
+      { text: 'Dependency inversion', correct: true },
+      { text: 'Delegates segragation' },
+    ],
+    comment: 'the <b>Dependency Inversion (one should "depend upon abstractions, [not] concretions.)</b> principle refers to a specific form of decoupling software modules.\
+     When following this principle, the conventional dependency relationships established from high-level, \
+     policy-setting modules to low-level, dependency modules are reversed, \
```

Comment:
Why do you add `\` to the end of line sometimes?

Reply:
read https://json5.org